src/accelerate/test_utils/testing.py

```diff
@@ -73,6 +74,21 @@ def get_backend():
 torch_device, device_count, memory_allocated_func = get_backend()
 
 
+@lru_cache
+def is_hub_online():
+    """
+    Checks if the huggingface_hub is online
+    """
+    response = requests.get("https://huggingface.co/api")
+    if response.status_code != 200:
```

Comment:
Technically, any 2** status code should be fine (though maybe 200 can be returned).

Also, I wonder if the URL is correct. I get 404 at this very moment but can still successfully download models from the hub. Is there maybe an official health check that could be called instead?